sys/include/net/gnrc/sixlowpan.h

```diff
@@ -11,11 +11,45 @@
  * @ingroup     net_gnrc
  * @brief       GNRC's 6LoWPAN implementation
  *
- * This module is for usage with the @ref net_gnrc_netapi.
+ * # Internal API and sub-modules
+ *
+ * Internally, @ref net_gnrc_sixlowpan is sub-divided into several sub-modules.
+ * This implement certain features of the 6LoWPAN standard. Currently
```

Comment:
typo, either:

- `this implement[s]`
- `[these] implement`

or for the latter variant: `each implementing ...`